app/views/welcome/payment.html.haml

```diff
@@ -1 +1,38 @@
-You owe me money!
\ No newline at end of file
+.uk-container.uk-text-justify
+
+  .uk-margin-top
+  
+  .uk-panel-hover
+    .uk-panel-title How to Make Payment
+    There are 3 simple ways to make payment for your order.
+    %br
+    %br
+    1) 
+    %strong Credit Card
+    %br    
+    We accept VISA and MasterCard. That's pretty much 95% of the credit cards out there.
+    %br
+    %br
+    %br
+    2) 
```

Comment:
(OPTIONAL ) instead of using 1) 2) etc.. please use <OL> <li>

this means ordered list it will automatically number your list